pkg/cloudprovider/providers/rackspace/rackspace.go

```diff
@@ -57,6 +64,11 @@ func (d *MyDuration) UnmarshalText(text []byte) error {
 	return nil
 }
 
+type MetaData struct {
```

Comment:
I don't think it matters, but the convention we use in gophercloud is `Metadata`. Might be worth being consistent, but it's really minor